main.go

```diff
@@ -32,7 +32,11 @@ func main() {
 		vargs.Template = defaultTemplate
 	}
 
-	client := NewClient(vargs.Room.String(), vargs.Token)
+	client := NewClient(
+		vargs.Url,
```

Comment:
`Url` should be `URL` https://github.com/golang/go/wiki/CodeReviewComments#initialisms